packages/components/src/tab-panel/test/index.tsx

```diff
@@ -34,7 +34,23 @@ const TABS = [
 
 const getSelectedTab = () => screen.getByRole( 'tab', { selected: true } );
 
+const originalGetClientRects = window.HTMLElement.prototype.getClientRects;
```

Comment:
Nit: should this be declared right before we override it in `beforeAll`? This would then be a `let` and we'd just set it in `beforeAll()`.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/46004/commits/3128c2684b703a8636e6af1c5a29aad01cde82eb